src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -47,16 +47,8 @@ public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clie
                 var request = new CodeWriterDeclaration("request");
                 var uri = new CodeWriterDeclaration("uri");
 
-                // Do note there is a subtle difference here, the lines are not identical
-                if (lowLevel)
-                {
-                    writer.Line($"var {request:D} = {GetPipeline(lowLevel)}.CreateRequest();");
-                }
-                else
-                {
-                    writer.Line($"var {message:D} = {GetPipeline(lowLevel)}.CreateMessage();");
-                    writer.Line($"var {request:D} = {message}.Request;");
-                }
+                writer.Line($"var {message:D} = {GetPipeline(lowLevel)}.CreateMessage();");
```

Comment:
FWIW - This generates a slightly more verbose implementation for the body of the LLC methods, but I think it's worthwhile to have `request` and `message` both be defined in both cases since it makes the implementation in this file much easier to reason about.